allensdk/test/brain_observatory/ecephys/stimulus_analysis/test_receptive_field_mapping.py

```diff
@@ -153,7 +153,7 @@ def test_threshold_rf(rf, threshold, expected_mask, expected_x, expected_y, expe
 
 @pytest.mark.parametrize('matrix,expected',
                          [
-                             (rf_field_real, (np.array([1.04991433e+05, 3.74217858e+00, 3.24465965e+00, 1.66477569e+00, 1.04485211e+00]), True)),
+                             (rf_field_real, (np.array([1.04990549e+05, 3.24465782e+00, 3.74217921e+00, 1.04486140e+00, 1.66478893e+00]), True)), # noqa
```

Comment:
I'm confused. If this was just an xy flip why do we need to change these values?

Reply:
The estimated x and y parameters from the gaussian_2d_fit are flipped, so the test expected values need to be flipped.

Reply:
Got ya. I can see the flip now. Missed it before due the way my browser was showing the data. Looks good.